sigs.yaml

```diff
@@ -174,6 +174,35 @@ sigs:
         teams:
         - name: kubevirt-community
           description: General Discussion
+workinggroups:
+    - dir: wg-arch-s390x
```

Comment:
@cfilleke @jschintag @vamsikrishna-siddu hey, here's the entry for the wg arch S390x, please chime in here with the things you want to have added, I'll happily integrate them

Reply:
/lgtm